src/components/TicketDetailModal/TicketDetailModal.tsx

```diff
@@ -0,0 +1,308 @@
+import {
+  AccessTime,
+  Add,
+  AddBox,
+  ArrowForward,
+  AttachFile,
+  Check,
+  Close,
+  CropLandscape,
+  FileCopy,
+  FormatListBulleted,
+  Inbox,
+  Label,
+  Notes,
+  Person,
+  Remove,
+  Share,
+  Style,
+  Title,
+  Visibility,
+  WorkOutline,
+} from "@material-ui/icons";
+import styled from "styled-components";
+
+import { GET_TICKET_BY_ID } from "../../api/tickets";
+import { ProjectContext } from "../../context/project";
+import { useFetch } from "../../hooks/useFetch";
+import { useTypeSafeContext } from "../../hooks/useTypeSafeContext";
+import TicketDefenition from "../../interfaces/Ticket";
+import Avatar from "../Avatar";
+import _Button from "../Button";
+import _Card from "../Card";
+import Dialog from "../Dialog";
+import EditableLabel from "../EditableLabel";
+import _EditableTicket from "../EditableTicket";
+import IconButton from "../IconButton";
+
+export interface TicketDetailModalProps {
+  isOpen: boolean;
+  ticketId: number | null;
+  onClose?: () => void;
+}
+
+export default function TicketDetailModal({ isOpen, ticketId, onClose }: TicketDetailModalProps): JSX.Element {
+  const project = useTypeSafeContext(ProjectContext);
+  const { data: ticketResponse } = useFetch<TicketDefenition>(GET_TICKET_BY_ID(project.id, ticketId!));
+
+  return (
+    <Dialog isOpen={isOpen} onClose={onClose} disableCloseButton>
+      <Card>
+        <Container>
+          <CloseIconButton onClick={onClose}>
+            <Close />
+          </CloseIconButton>
+
+          <Headline>
+            <Title />
+
+            <TicketTitleEditableLabel value={ticketResponse?.name} />
+          </Headline>
+
+          <Content>
+            <Main>
+              <Paragraph>
+                <ParagraphTitle>
+                  <Notes />
+                  <MainTitle>Description</MainTitle>
+                </ParagraphTitle>
+
+                <DescriptionForm>
+                  <DescriptionFormContainer>
+                    <EditableDescription
+                      placeholder="Add a more detailed description…"
+                      value={ticketResponse?.description}
+                      readOnly
+                    />
+
+                    <DescriptionButtons>
+                      <DescriptionSaveButton title="Save" />
+                      <Button title="Cancel" />
+
+                      <Button title="Formatting help" />
+                    </DescriptionButtons>
+                  </DescriptionFormContainer>
+                </DescriptionForm>
```

Comment:
I am going to replace this.

Reply:
What are you going to replace this with?

Reply:
Check how it is in Trello app

Reply:
What is wrong? I'm just asking you what you mean by your sentence.